frontend/src/model/Tile.ts

```diff
@@ -41,6 +46,24 @@ class Tile {
       right: prevEdges.bottom,
       top: prevEdges.right,
     };
+    switch (this.rotation) {
+      case 0: {
+        this.rotation = 270;
+        break;
+      }
+      case 90: {
+        this.rotation = 0;
+        break;
+      }
+      case 180: {
+        this.rotation = 90;
+        break;
+      }
+      default: {
+        this.rotation = 180;
+        break;
+      }
+    }
```

Comment:
It'd be shorter if we subtract 90 form rotation, (but before this operation check if this.rotation === 0 and then subtract from 360